server/websocket.go

```diff
@@ -36,6 +36,10 @@ const (
 )
 
 func (p *Plugin) handleClientMessageTypeScreen(us *session, msg clientMessage, handlerID string) error {
+	if cfg := p.getConfiguration(); cfg == nil || cfg.AllowScreenSharing == nil || !*cfg.AllowScreenSharing {
```

Comment:
Does this basically mean the default is to not allow screensharing? (although probably the first two checks are not expected to ever happen)

Reply:
Sounds good to me.